### PR TITLE
Added third paragraph

### PR DESCRIPTION
**Pull request by duser1278**
This is third paragraph

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -7,6 +7,7 @@ <h1>My First Heading</h1>
 
 <p>My First paragraph.</p>
 <p>My secod paragraph.</p>
+<p>My Third paragraph.</p>
```
Comment (learnigbyganesh) [inline index.html:10]: what is purpose of this?

Comment (duser1278) [reply index.html:10]: this is demo purpose

Comment (learnigbyganesh) [reply index.html:10]: ok

**Review by learnigbyganesh — APPROVED** [review]
approved

**Pull request merged**